Concept Jero

Jair0 opened the pull request:
Gameplay by Jero

jeronimo-schreyer committed:
Primeros movimientos del player

Esta todo puesto en cualquier lado. A medida que avancemos refactoreamos
un poco

Jair0 commented:
DONT MERGE
In progress

jeronimo-schreyer committed:
Primeros movimientos del player

Faltaban estos archivos

jeronimo-schreyer committed:
Refactor y Feature

Cambie el procesamiento del input para que ahora forme parte de la
tenebrosa funcion Update
Ademas agregue salto variable

jm42 commented:
Lo veo así.

![jero-r1](https://cloud.githubusercontent.com/assets/3297150/15631204/b73899c8-2532-11e6-9b07-3e590f258ea2.png)

La consola dice:

```
THREE.WebGLRenderer 77 three.min.js:629:52
THREE.WebGLRenderer: .shadowMapEnabled is now .shadowMap.enabled. three.min.js:807:136
THREE.WebGLRenderer: .shadowMapType is now .shadowMap.type. three.min.js:807:319
Error: WebGL: Exceeded 16 live WebGL contexts for this principal, losing the least recently used one. three.min.js:632:219
false
```

jeronimo-schreyer committed:
Primer Comportamiento de camara

Ademas saque jumpTime del scope de la escena y lo agregue al userData
del player

jeronimo-schreyer committed:
Lerpeando el movimiento de la camara

jeronimo-schreyer committed:
Fixeando la camara al llegar a un lugar

para ilustrar este comportamiento, hay un proto de lo que podria ser un
nivel
